packages/mds-api-server/middleware/api-error-handling.ts

```diff
@@ -1,15 +1,23 @@
 import logger from '@mds-core/mds-logger'
 import { ErrorCheckFunction, isServiceError } from '@mds-core/mds-service-helpers'
-import { AuthorizationError, BadParamsError, ConflictError, NotFoundError, ValidationError } from '@mds-core/mds-utils'
+import {
+  AuthorizationError,
+  BadParamsError,
+  ConflictError,
+  DependencyMissingError,
+  NotFoundError,
+  ValidationError
+} from '@mds-core/mds-utils'
 import type express from 'express'
-import HttpStatus from 'http-status-codes'
+import { StatusCodes } from 'http-status-codes'
```

Comment:
`HttpStatus` is deprecated, doesn't have all codes on it, even. `StatusCodes` is complete, however.